Appraisals

```diff
@@ -17,3 +17,9 @@ appraise 'avro1_11-rails7_1' do
   gem 'activesupport', '~> 7.1.1'
   gem 'activemodel', '~> 7.1.1'
 end
+
+appraise 'avro1_11-rails7_2' do
```

Comment:
Can you add this new gemfile to the build [matrix](https://github.com/salsify/avromatic/blob/master/.circleci/config.yml#L69)?

Reply:
Not a problem, done 👍